src/DataFrames.jl

```diff
@@ -99,6 +108,6 @@ include("dataframerow/show.jl")
 include("abstractdataframe/sort.jl")
 include("dataframe/sort.jl")
 
-include("deprecated.jl")
+# include("deprecated.jl")
```

Comment:
commenting this out was done for testing purposes... will be restored, please ignore